src/frontend/src/features/utils/task-import-parser/index.ts

```
@@ -0,0 +1,46 @@
+    #serialize(taskId: string) {
+        const expiresAt = Date.now() + this.#staleTime;
+        return `${taskId}:${expiresAt}`;
+    }
+
+    #deserialize(value: string) {
+        const [taskId, expiresAt] = value.split(":");
+        return { taskId, expiresAt: parseInt(expiresAt) };
+    }
+
+    get() {
+        if (typeof window === undefined || !this.mailboxId) return null;
```

Biome (2.1.2)

[error] Invalid typeof comparison.

Compare with one of the following string literals:


(lint/correctness/useValidTypeof)